service/raft/raft_group_registry.hh

```diff
@@ -24,6 +23,9 @@ namespace service {
 
 class raft_rpc;
 class raft_sys_table_storage;
+template <typename Clock> class raft_address_map_t;
+using raft_address_map = raft_address_map_t<seastar::lowres_clock>;
+using raft_ticker_type = seastar::timer<lowres_clock>;
```

Comment:
This declaration is repeated a couple of times, maybe put it in raft_address_map_fwd.hh

Reply:
I am fine repeating it the 3 times I have to. I think having a header file with just 3 lines to avoid 3 repetitions in it is an overkill. Back then opening a file when compiling is expensive. Please let me know if you insist.

Reply:
What kind of overhead do you mean? It shouldn't affect compilation time in a noticeable way.

There is a precedent in utils/preempt.hh

Reply:
There is no need to in the latest version, forward declaration works.